### PR TITLE
Jezz fix invite users

### PR DESCRIPTION
**Pull request by JeremiahPinto**

### CODE REVIEW

**Comment by fezproof** [comment]
No description
What did you do?
What are the commits?
*I expect better*

**Review by fezproof — CHANGES_REQUESTED** [review]
*Tell me what you are trying to do*

File: src/components/admin/StaffCard.vue [hunk 1]
```diff
@@ -74,48 +78,68 @@ export default {
...
+    }).then((response) => {
+      const regionIds = response.data.map(regionId => regionId.region);
+      this.findRegions({
+        query: {
+          _id: {
+            $in: regionIds,
+          },
+        },
+      });
+      this.finished = true;
```
Comment (fezproof) [inline src/components/admin/StaffCard.vue:108]: What is this for?
Cant you do anything with the inbuilt booleans?

File: src/components/admin/StaffCard.vue [hunk 2]
```diff
@@ -1,6 +1,10 @@
 <template>
   <v-card flat>
-    <v-list v-show="!isFindPending" three-line subheader>
```
Comment (fezproof) [inline src/components/admin/StaffCard.vue:3]: Remove all these changes, this is insane scope creeping
Please recreate changes in a seperate PR
*I expect better*

**Review by fezproof — DISMISSED** [review]

File: src/components/admin/StaffCard.vue (hunk 1 above)
Comment (JeremiahPinto) [reply src/components/admin/StaffCard.vue:108]: The computed prop is called before the data is received from the getters which prints a bunch of console errors. This prevents that

**Comment by JeremiahPinto** [comment]
The main reason for this PR was to fix a bug where the user could press the enter key without having permissions set and the response would still be sent. The other changes happened to be there coz i forgot to create a new branch

**Review by fezproof — APPROVED** [review]

**Pull request merged**